js/ui/camera.js

```diff
@@ -411,7 +413,8 @@ class Camera extends Evented {
      * details not specified in `options`.
      *
      * @memberof Map#
-     * @param {CameraOptions|AnimationOptions} options Options describing the destination and animation of the transition.
+     * @param {Object} options Options describing the destination and animation of the transition.
+    *            Accepts [CameraOptions](#CameraOptions) and [AnimationOptions](#AnimationOptions).
```

Comment:
Thanks. Should have known you'd be following a standard :)

(I actually found that Flow doc page a bit ambiguous, but the page on [disjoint unions](https://flowtype.org/docs/disjoint-unions.html#_) makes clear that unions are non-disjoint by default)